frame/support/src/lib.rs

```diff
@@ -105,13 +105,38 @@ macro_rules! parameter_types {
 		$( #[ $attr:meta ] )*
 		$vis:vis const $name:ident: $type:ty = $value:expr;
 		$( $rest:tt )*
+	) => (
+		$( #[ $attr ] )*
+		$vis struct $name;
+		$crate::parameter_types!{IMPL_CONST $name , $type , $value}
+		$crate::parameter_types!{ $( $rest )* }
+	);
+	(
+		$( #[ $attr:meta ] )*
+		$vis:vis $name:ident: $type:ty = $value:expr;
+		$( $rest:tt )*
 	) => (
 		$( #[ $attr ] )*
 		$vis struct $name;
 		$crate::parameter_types!{IMPL $name , $type , $value}
 		$crate::parameter_types!{ $( $rest )* }
 	);
 	() => ();
+	(IMPL_CONST $name:ident , $type:ty , $value:expr) => {
+		impl $name {
+			pub fn get() -> $type {
+				$value
+			}
+			pub const fn get_const() -> $type {
+				$value
+			}
```

Comment:
```suggestion
			pub const fn get() -> $type {
				$value
			}
```

Reply:
What is the reason to have a dedicated `get_const`?

Reply:
> In per_thing we defined one fn one on the trait and one const fn one on the impl and the compiler was smart enough to know which one is which. I cannot do the same, here, hence named get_const().

Reply:
Or maybe we can implement _only_ a `const fn get` for when we have the `const` token.